src/pages/components/input.mdx

```diff
@@ -123,6 +123,8 @@ import IconsCode from '!!raw-loader!../../examples/forms/inputs/MediaInput.tsx';
 ### Placeholder
 
 Placeholder text provides the user with example content or additional context.
+Use icon to provide a visual affordance when the [label is hidden](#hidden-label)
```

Comment:
[nit] wondering if this should be: `Use [an] icon to provide visual affordance...`